toolbox/src/DiscreteFilter.cpp

```diff
@@ -116,6 +118,11 @@ bool DiscreteFilter::initialize(BlockInformation* blockInfo, wbt::Error* error)
     // Create the filter object
     // ========================
 
+    // Get the width of the signals
+    unsigned inputSignalWidth  = blockInfo->getInputPortWidth(INPUT_IDX_SIGNAL);
+    unsigned outputSignalWidth = blockInfo->getOutputPortWidth(OUTPUT_IDX_SIGNAL);
+    assert(inputSignalWidth == outputSignalWidth);
```

Comment:
I am not an expert of Simulink, but what happens with this assert if the user connect two signal of different size to the input and output of the block? Simulink crashes?